src/components/Modal/Modal.css

```diff
@@ -0,0 +1,88 @@
+.modal {
```

Comment:
Do we need this CSS in our webapp? is it not there in the Doppler-UI-Library?

Reply:
😴 💤 💤

Reply:
jaja banca que estaba durmiendo. Ya lo corregi, solo que no subi el pr porque me queda hacer el send aun

Reply:
Jaja, en realidad sólo quería una respuesta acerca de los css, para entender cómo lo vamos a trabajar. Luego reviso los cambios a ver si lo infiero de ahí, sino luego te vuelvo a consultar.